plugin/password.lua

```diff
@@ -0,0 +1,54 @@
+
+local default_password = minetest.settings:get("default_password")
+if not default_password then
+	-- Do not use if default password is not set
+	return
+end
+
+local only_new_players = minetest.settings:get("beerchat.password.mode") == "new"
+local message = minetest.settings:get("beerchat.password.message") or "*** Please change your password"
+
+local password_notify = {}
+local auth_handler = minetest.get_auth_handler()
```

Comment:
Is it safe to keep instance in lua space?
Will this cause problems if another mod registers authentication handler after mods are loaded?